collectors.rb

```diff
@@ -57,6 +57,34 @@ def collect_ganeti(cluster)
     @node_cache.write
   end
 
+  def collect_chef(url, client, key)
+    chef_template = ERB.new File.new('cache_templates/chef_template.erb').read,
+                            nil, '%'
+
+    ChefAPI::Connection.new(endpoint: url, client: client, key: key) do |connection|
+      connection.nodes.each do |n|
+        node_name = n.name || 'unknown'
+        ram_total = n.automatic['memory']['total'] || 'unknown'
+        ram_free = n.automatic['memory']['free'] || 'unknown'
+        cpus_total = n.automatic['cpu']['total'] || 'unknown'
+        cpus_real = n.automatic['cpu']['real'] || 'unknown'
+
+        # Collect disk size
+        disk_size = disk_usage = disk_count = 0
+        n.automatic['filesystem2']['by_device'].each do |_key, device|
+          next unless device['fs_type'] == 'ext4'
```

Comment:
ext3/4 are the most likely to have relevant data, let's check those two, unless someone from infra says there are others.